tests/stubs/devices/mockserver.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2019 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package devices
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/paypal/gatt"
+	"github.com/paypal/gatt/examples/option"
+
+	"github.com/kubeedge/kubeedge/tests/stubs/devices/services"
+)
+
+func createServiceAndAdvertise(d gatt.Device, s gatt.State) {
+	// Setup GAP and GATT services for Linux implementation.
+	// OS X doesn't export the access of these services.
```

Comment:
Pls remove all comments relevant to OS X since we are currently not providing OS X support

Reply:
Updated